base/src/main/java/com/smartdevicelink/proxy/rpc/SubscribeVehicleDataResponse.java

```diff
@@ -454,31 +454,31 @@ public VehicleDataResult getTurnSignal() {
 
     /**
      * Sets electronicParkBrakeStatus
-     * @param electronicParkBrakeStatus
+     * @param electronicParkBrakeStatus a VehicleDataResult related to the electronic park brake status
      */
     public void setElectronicParkBrakeStatus(VehicleDataResult electronicParkBrakeStatus){
         setParameters(KEY_ELECTRONIC_PARK_BRAKE_STATUS, electronicParkBrakeStatus);
     }
 
     /**
      * Gets electronicParkBrakeStatus
-     * @return VehicleDataResult
+     * @return a VehicleDataResult related to the electronic park brake status
      */
     public VehicleDataResult getElectronicParkBrakeStatus(){
         return (VehicleDataResult) getObject(VehicleDataResult.class, KEY_ELECTRONIC_PARK_BRAKE_STATUS);
     }
 
     /**
      * Sets cloudAppVehicleID
-     * @param cloudAppVehicleID a boolean value
+     * @param cloudAppVehicleID a VehicleDataResult related to the cloud app vehicle ID
      */
     public void setCloudAppVehicleID(VehicleDataResult cloudAppVehicleID){
         setParameters(KEY_CLOUD_APP_VEHICLE_ID, cloudAppVehicleID);
     }
 
     /**
-     * Gets a boolean value. If true, means the cloudAppVehicleDataID data has been subscribed.
-     * @return a Boolean value.
+     * Gets a boolean value. If true, means the cloudAppVehicleDataID data has been unsubscribed.

```

Comment:
This description is for another method because this method returns `VehicleDataResult`